### ISSUE TITLE
pkg installation conflict after synth build with FLAVOR

### ISSUE THREAD

**Issue by daemonblade** [jrmarino/synth#106]
The optional package installation after a synth build doesn't know about **flavoured** names. eg:
```
# synth build devel/py-virtualenvwrapper@py27
[building]
The task is complete.  Final tally:
Initial queue size: 6
    packages built: 6
           ignored: 0
           skipped: 0
            failed: 0

Duration: 00:00:42
The build logs can be found at: /var/log/synth
Would you like to rebuild the local repository (Y/N)? y
Stand by, recursively scanning 1 port serially.
Packages validated, rebuilding local repository.
Local repository successfully rebuilt
Would you like to upgrade your system with the new packages now (Y/N)? y
Updating Synth repository catalogue...
Fetching meta.txz: 100%    260 B   0.3kB/s    00:01
Fetching packagesite.txz: 100%  157 KiB 160.5kB/s    00:01
Processing entries: 100%
Synth repository update completed. 667 packages processed.
All repositories are up to date.
pkg: No packages available to install matching 'devel/py-virtualenvwrapper@py27' have been found in the repositories
Unfortunately, the system upgraded failed.
```

**Comment by daemonblade** [comment]
Incidentally, thank you very much for getting FLAVOR working on _synth_ so quickly after its introduction onto the ports tree!

**Comment by jrmarino** [comment]
thanks for the report.  I have to get *synth status* working first, then I'll look at this (and check out report on build list file not working right)

**Comment by jrmarino** [comment]
Hopefully this is fixed with v2.01 now.
Reopen if this is still a problem.
